src/utility/note-functions.js

```diff
@@ -0,0 +1,108 @@
+import axios from "axios";
+import { v4 as uuid } from "uuid";
```

Comment:
suggestion: avoid this package, and write your own function to generate unique number